components/collection/drop/HolderOfGenerative.vue

```diff
@@ -253,27 +246,22 @@ const mintNft = async () => {
     }
 
     initTransactionLoader()
-    const cb = api.tx.utility.batchAll
-    const mint = api.tx.nfts.mint(
+    const cb = api.tx.nfts.mint
+
+    const args = [
       collectionId.value,
       collectionRes.items,
```

Comment:
@Jarsen136 @vikiival should we revert this section on paid drops component until this issue is resolved? https://github.com/kodadot/nft-gallery/issues/9328

Reply:
#9328 has nothing to do with a on-chain BadWitness error.
The implementation is correct.